### PR TITLE
Referencing issue #465 Create custom error pages for 403, 404,500

### PR DESCRIPTION
**Pull request by swekshajha12**
#465 I added the templates for error 403 , 404 and 500 in vms/volunteer and did the corresponding changes in views.py and url. Kindly provide your reviews.

### CODE REVIEW

**Comment by swekshajha12** [comment]
Am I only supposed to create the pages?

**Comment by swekshajha12** [comment]
Sure , I'll do it ! Does it need to be exactly same as https://github.com/systers/portal/pull/225?
I'll cleanup everything else!

**Comment by tapasweni-pathak** [comment]
Yes, we will keep both Systers Portal and Systers VMS similar.

File: vms/volunteer/views.py [hunk 1]
```diff
@@ -54,10 +55,10 @@ def delete_resume(request, volunteer_id):
                 return HttpResponseRedirect(reverse('volunteer:profile', args=(volunteer_id,)))
             except:
                 raise Http404
+             
     else:
         return HttpResponse(status=403)
-
-'''
+'''
```
Comment (tapasweni-pathak) [inline vms/volunteer/views.py]: Can you please run flake8 on your changes?

Comment (swekshajha12) [reply vms/volunteer/views.py]: flake

I get this on running flake8 , shall I go do all these changes?

Comment (tapasweni-pathak) [reply vms/volunteer/views.py]: Yes.

Comment (tapasweni-pathak) [reply vms/volunteer/views.py]: Please let me know if you are stuck on something for long, we can see it together. Are you still working on this?

Comment (swekshajha12) [reply vms/volunteer/views.py]: Yes , I'm doing the flake8 changes .

Comment (swekshajha12) [reply vms/volunteer/views.py]: Is this pr still valid? Can I push the commits with Flake8 chnages?

Comment (tapasweni-pathak) [reply vms/volunteer/views.py]: If this is the final one, you need to squash your commits.

Comment (swekshajha12) [reply vms/volunteer/views.py]: I created a cleaner pr with the suggested changes here #513. So closing this one.

**Pull request closed without merging**